Sources/NIOPosix/SelectorWSAPoll.swift

```diff
@@ -102,7 +104,7 @@ extension Selector: _SelectorBackendProtocol {
             }
         } else {
             let result = self.pollFDs.withUnsafeMutableBufferPointer { ptr in
-                WSAPoll(ptr.baseAddress!, UInt32(ptr.count), time)
+                WSAPoll(ptr.baseAddress!, UInt32(ptr.count), 1)
```

Comment:
I'm not entirely following why this change needed to be made. Can you elaborate this?

Reply:
Why is WSAPoll not being made aware of the change? That seems like the obvious bug.

Reply:
Oh that's clever! I'll take a look at that

Reply:
We need the eventFD pattern here. Sadly. QueueUserAPC should be thrown out.